src/main/java/frc/robot/commands/climber/Climb.java

```diff
@@ -35,13 +36,17 @@ public Climb(DoubleSupplier speed, Climber climber, Intake intake) {
 
   @Override
   public void initialize() {
-    m_timer.start();
-    m_intake.setExtended(true);
+    if (m_intake.isExtended()) {
+      m_isIntakeExtended = true;
+    } else {
+      m_timer.start();
+      m_intake.setExtended(true);
+    }
   }
 
   @Override
   public void execute() {
-    if (m_intake.isExtended()) {
+    if (m_isIntakeExtended) {
```

Comment:
'
if (m_isIntakeExtended) {
  m_climber.set(m_speed.getAsDouble());
} else if (m_timer.get() > 0.5) {
  m_climber.setPiston(true);
  m_climber.set(m_speed.getAsDouble);
} else {
  m_climber.stop();
}
'

Reply:
Resolved in commit 408960c